update_mgmt_pom.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
```

Comment:
Lets avoid having scripts in the root. We should put them under the eng folder.

Reply:
will do...